tests/test_return_code.py

```diff
@@ -2,69 +2,51 @@
 
 import os
 import subprocess
+from functools import partial
+from typing import List
 
 DIRTY_NOTEBOOK = os.path.join("tests", "data", "notebook_for_testing.ipynb")
 CLEAN_NOTEBOOK = os.path.join("tests", "data", "clean_notebook.ipynb")
```

Comment:
This could be rewritten with `Path` and a `TEST_DATA_DIR` variable to make it more concise.